apiserver/apiserver.go

```diff
@@ -625,7 +633,8 @@ func (srv *Server) endpoints() []apihttp.Endpoint {
 		noModelUUID     bool
 	}
 	var endpoints []apihttp.Endpoint
-	controllerModelUUID := srv.shared.statePool.SystemState().ModelUUID()
+	systemState, _ := srv.shared.statePool.SystemState()
```

Comment:
another error to process, though I'm not sure why the same method is calling srv.shared.statePool.SystemState() again at ln 769